web-app/src/components/SettingsDialog.js

```diff
@@ -478,12 +478,44 @@ class SettingsDialog extends React.Component {
 									style={{
 										color: "rgba(40, 26, 26, 0.56)",
 										marginTop: "16px",
-										padding: "16px 0px",
+										padding: "16px 0px 0px 0px",
 									}}
 								>
-									Custom nation colours
+									Nation colours
+								</Typography>
+								<div width="100%">
+									<FormControlLabel
+										style={{
+											width: "100%",
+											maxWidth: "920px",
+											paddingLeft: "0px",
+										}}
```

Comment:
Use the styles object as described below :). Also this 920px value appears a few times. Maybe pull it out into a variable at the top of the script?

Reply:
Yes, that's a good idea, but that would be across the app - a "maxWidth" variable that we can address everywhere. Where could I define that?

Reply:
I added this, but would like to make it app wide.

Reply:
Yep I'll make it app wide in a future pr. Slightly tricky but not too bad

Reply:
There are other elements that have to be set app-wide, like the primarycolor, right? Isn't that just setting it in app.js?